packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx

```diff
@@ -431,17 +416,40 @@ export const Account = ({
               ))}
             </>
           ) : (
-            <li
-              className={`${styleClasses.borderInfoDisplay} ${styleClasses.li}`}
-            >
+            <li className="account-li account-border-info">
               This account has nothing attached.
             </li>
           )}
         </Guard>
+
+        {/* Temporary check for fake hard-coded value until we fetch actual subscriptions in FXA-4237 */}
+        {subscriptions &&
+          subscriptions[0].productName !== 'Cooking with Foxkeh' && (
+            <>
+              <li className="account-li">
+                <h3 className="account-header">Subscriptions</h3>
+              </li>
+              {subscriptions && subscriptions.length > 0 ? (
+                <>
+                  {subscriptions.map((subscription) => (
+                    <Subscription
+                      key={subscription.subscriptionId}
+                      {...subscription}
+                    />
+                  ))}
+                </>
+              ) : (
+                <li className="account-li account-border-info">
+                  This account doesn't have any subscriptions.
+                </li>
+              )}
+            </>
+          )}
       </ul>
+
       <hr className="border-grey-50 mb-4" />
-      <h3 className="mt-0 my-0 mb-1 text-lg">Account History</h3>
-      <div className={styleClasses.borderInfoDisplay}>
+      <h3 className="account-header">Account History</h3>
+      <div className="account-li account-border-info">
```

Comment:
Thanks again for cleaning these up! I wasn't a fan of the styleClasses.$thing move.